### ISSUE TITLE
Document farmOS-client and farmOS-native

### ISSUE THREAD

**Issue by mstenta** [farmOS-legacy/farmOS.org#27]
https://github.com/farmOS/farmOS-client

https://github.com/farmOS/farmOS-native

**Comment by mstenta** [comment]
https://github.com/farmOS/farmOS-native/issues/19#issuecomment-382198804

**Comment by mstenta** [comment]
https://github.com/farmOS/farmOS-native/issues/14

**Comment by jgaehring** [comment]
Is there somewhere we could possibly just start a markdown document, create some headers like "Architecture", "Dev Env", "API Requests", "Authentication", "Code Quality (Testing & Linting)", etc, and then we could start by putting the links to those issues/comments under those headers? That way we won't lose track of them until we have a chance to properly integrate their content.

Would this repo be a good place for something like that? I don't think pinning such a document to either the native or client repo is a good idea, since there will be equal amounts of info that is relevant to one or the other or both, and it would be nice to everything in one place. There may also be some API documentation which could be relevant to other projects.

**Comment by mstenta** [comment]
Yes definitely - I think that's a great idea - I'll start a branch...

> I don't think pinning such a document to either the native or client repo is a good idea, since there will be equal amounts of info that is relevant to one or the other or both, and it would be nice to everything in one place.

Yea agreed, the farmOS.org repo is a nice central place for all this related info.  I'll make a new page in the "Development" section.

**Comment by mstenta** [comment]
Pushed a commit that just creates the outline: https://github.com/farmOS/farmOS.org/compare/client-native

**Comment by mstenta** [comment]
Branch is called "client-native".

**Comment by jgaehring** [comment]
@mstenta , do we want to close this issue, since we've got the groundwork laid now, and it's maybe a little too broad at this point to effectively track smaller documentation issues as they arise? It looks like all the links in above comments have been incorporated to some degree, even if they will still need more fleshing out.

Hope it's ok I just opened a smaller issue, #29 , to track the documentation of the Architecture more specifically.
